examples/mir_demo_server/generated/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+set(PROTOCOL "${CMAKE_CURRENT_SOURCE_DIR}/../protocol/server-decoration.xml")
+
+add_custom_command(
+    OUTPUT ${CMAKE_CURRENT_SOURCE_DIR}/server-decoration_wrapper.cpp
+    OUTPUT ${CMAKE_CURRENT_SOURCE_DIR}/server-decoration_wrapper.h
+    DEPENDS mir_wayland_generator
+    DEPENDS ${PROTOCOL}
+    COMMAND "sh" "-c" "${CMAKE_BINARY_DIR}/bin/mir_wayland_generator org_kde_kwin_ ${PROTOCOL} header >${CMAKE_CURRENT_SOURCE_DIR}/server-decoration_wrapper.h"
+    COMMAND "sh" "-c" "${CMAKE_BINARY_DIR}/bin/mir_wayland_generator org_kde_kwin_ ${PROTOCOL} source >${CMAKE_CURRENT_SOURCE_DIR}/server-decoration_wrapper.cpp"
+)
```

Comment:
I think I wrote a macro in the other generated code CMakeLists that makes all this less repetitive and error prone. Might want to use something like that if possible.